modules/diskq/logqueue-disk-non-reliable.c

```diff
@@ -152,7 +152,7 @@ _write_message_to_disk(LogQueueDisk *self, LogMessage *msg)
 {
   gboolean consumed = FALSE;
 
-  if (qdisk_started(self->qdisk) && qdisk_is_space_avail(self->qdisk, 64))
```

Comment:
Question:
This with the hard-coded value `64` can be considered as a hint, so we can skip message serialization and its cost when the disk-buffer has no "minimum" space at all.

Can't we add this to your new hint function?

Reply:
Thanks, added!